### DISCUSSION TITLE
Welcome to GFDL_atmos_cubed_sphere Discussions!

### DISCUSSION THREAD

**Discussion by laurenchilutti** [NOAA-GFDL/GFDL_atmos_cubed_sphere#168] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lharris4** [comment]
Hi everyone! I am Lucas Harris, the head of the FV3 Team and the Deputy Division Lead of GFDL's Weather and Climate Dynamics Division. My primary interest are in dynamical core development, focusing on variable-resolution and convective-scale modeling. I have joined this community to bring together the worldwide community of FV3-based models for mutual benefit, and to allow our innovations and understanding to spread throughout the world.
